App/View Controllers/Posts/PostsPageViewController.swift

```diff
@@ -1016,6 +1016,34 @@ final class PostsPageViewController: ViewController {
                     quotePost()
                 }
             }))
+        } else {
+            items.append(IconActionItem(.copyPost, block: {
+                func copyPost(_ post: Post) {
```

Comment:
I'm not sure why this is its own function that we immediately call. Could we do all the things directly in the action block?

Reply:
So I have this idea for a future feature where you'd be able to enter a "select mode" that would display all currently loaded posts into a grid of thumbnails.

From there you could either:
1) Navigate quickly by jumping to a post
2) Flag posts as selected and either copy them as quotes or
3) Save a screenshot of selected posts stitched together

So I thought it might be useful to have as a function that other things could also call.

But I'm happy to put into the action block for now and make it a function when actually needed

Reply:
This should now be correct in latest commit.
Will pull it out into an accessible function if/when the need arises